.github/ISSUE_TEMPLATE/problem.yml

```diff
@@ -0,0 +1,115 @@
+name: Problem
+description: Create a report to help us improve
+labels: [bug, triage]
+
+body:
+  - type: textarea
+    id: problem
+    attributes:
+      label: Describe the problem
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+
+  - type: textarea
+    id: template
+    attributes:
+      label: Template
+      description: Many problems are related to a specific Copier template. If you can provide a template, please do it. It can be a Git repo URL, a .zip file containing the template, or instructions to build it.
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. Run `copier ...`
+        2. Answer ...
+        3. Choose ...
+        4. See error
+    validations:
+      required: true
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: Logs
+      description: Provide logs of the problem. These will typically happen when getting to the latest step explained above if Copier produces an error. It helps too if you paste your copying session.
+      render: console
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behavior
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+
+  - type: textarea
+    id: additional_info
+    attributes:
+      label: Screenshots/screencasts/logs
+      description: If applicable, add screenshots/screencasts/logs to help explain your problem.
+
+  - type: dropdown
+    id: env-os
+    attributes:
+      label: Operating system
+      options:
+        - macOS
+        - Linux
+        - Windows
+    validations:
+      required: true
+
+  - type: input
+    id: env-os-distro
+    attributes:
+      label: Operating system distribution and version
+      description: E.g. Ubuntu 22.04
+    validations:
+      required: true
+
+  - type: input
+    id: env-copier
+    attributes:
+      label: Copier version
+      description: |
+        Please run the following command and copy the output below:
+
+        ```shell
+        copier --version
+        ```
+
+    validations:
+      required: true
+
+  - type: input
+    id: env-python
+    attributes:
+      label: Python version
+      description: E.g. CPython 3.11
+    validations:
+      required: true
+
+  - type: dropdown
+    id: env-install
+    attributes:
+      label: Installation method
+      options:
+        - pipx+pypi
+        - pipx+git
+        - pip+pypi
+        - pip+git
+        - local build
+        - distro package
+    validations:
+      required: true
+
+  - type: textarea
+    id: context
+    attributes:
+      label: Additional context
+      description: Add any other context about the problem here.
```

Comment:
I think we should have a command like: `copier env-info` that provides all required info.

Then, I'd start the form by asking this info. The user just needs to copy & paste.

It'd be extra nice if the results of the env info and logs could get collapsed inside a `<details>` block when producing the issue.

Reply:
> I think we should have a command like: `copier env-info` that provides all required info.
> 
> Then, I'd start the form by asking this info. The user just needs to copy & paste.

I was about to suggest the same thing, only with a different subcommand name. For instance, DVC uses [`dvc doctor`](https://dvc.org/doc/command-reference/doctor#doctor) for reporting environment info in [their issue template](https://github.com/iterative/dvc/blob/1f03c40dbe47af8a7f64f6e1200aecd4b07273b0/.github/ISSUE_TEMPLATE/bug_report.md?plain=1#L45-L64). But `copier env-info` is fine, too. :+1: I'll work on a PR in the main repo, and when that's merged and released, we can update the issue template accordingly.

> It'd be extra nice if the results of the env info and logs could get collapsed inside a `<details>` block when producing the issue.

Do you mean the output should be contained in a `<details>` block already? It seems that GitHub issue forms don't support collapsible sections natively, otherwise we could mark the env info section as collapsible instead of relying on a reporter to copy the info _with the `<details>` block_.

Reply:
Doctor sounds cool! 

![](https://media.giphy.com/media/ZoBn8lxRuCX9m/giphy-downsized-large.gif)